README.md

```diff
@@ -460,3 +492,13 @@ You are welcomed to fork the project and make pull requests.
  * [ ] Legend gradient transformation (if we used the log scale)
  * [ ] Have multiple legends depending on the attribute
  * [ ] Legend marker radius
+
+
+## Release History
+* Version 1.0.0 ?
```

Comment:
thanks! i updated the readme.md with that info and as well added a bit of info i thought might be helpful for first time dev like me.  lemme know if it's too verbose!